graaljs/graaljs-spring-boot-d3/src/test/java/com/example/GraaljsSpringBootD3ApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package com.example;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class GraaljsSpringBootD3ApplicationTests {
+
+	@Test
+	void contextLoads() {
+	}
```

Comment:
Please implement a simple test case that runs `generateGraph`. For inspiration, here's the test from the Photon demo: https://github.com/graalvm/graal-languages-demos/blob/0951909cb64a04262eaa6a27bc5800846e785019/graalwasm/graalwasm-spring-boot-photon/src/test/java/com/example/demo/DemoApplicationTests.java#L21